Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/auditrepository/hibernate/AuditRepositoryDAOTest.java

```diff
@@ -24,8 +24,10 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package gov.hhs.fha.nhinc.hibernate;
+package gov.hhs.fha.nhinc.auditrepository.hibernate;
 
+import gov.hhs.fha.nhinc.auditrepository.hibernate.AuditRepositoryRecord;
+import gov.hhs.fha.nhinc.auditrepository.hibernate.AuditRepositoryDAO;
```

Comment:
There are two added imports, but no lines added in this PR that use them.  Can they be removed?